### ISSUE TITLE
[A11Y] Add lang attribute to the <html> element

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5893]
**Describe the bug**

The `HTML` tag is missing the `lang` attribute. The attribute should be added, and should match the currently selected language code (not locale).

**To Reproduce**
1. Go to any page
2. Right-click the page and select "View page source"
3. Observe that there is no `lang` attribute on the `<html>` element

**Expected behavior**

Expected to see `lang` attribute on the `<html>` element.

**Additional context**

- https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html
- https://talk.getindico.io/t/language-page-language-attribute-s/3206
